tools/translations/ru.po

```diff
@@ -734,7 +734,7 @@ msgstr "Нет совпадений"
 
 #: tools/editor/code_editor.cpp
 msgid "Replaced %d Ocurrence(s)."
-msgstr "Заменено %d совпадений."
+msgstr "Заменено совпадений: %d."
```

Comment:
You seem to undo some changes that were made by @toby3d in 8f6b5ffd, is it on purpose, or a merge mistake?